examples/todo-list/src/__tests__/integration/todo-list-image.integration.ts

```diff
@@ -0,0 +1,62 @@
+import {expect, toJSON} from '@loopback/testlab';
+import {TodoListApplication} from '../../application';
+import {TodoListImageRepository, TodoListRepository} from '../../repositories';
+import {
+  givenRunningApplicationWithCustomConfiguration,
+  givenTodoListImageInstance,
+  givenTodoListInstance,
+  givenTodoListRepositories,
+} from '../helpers';
+
+describe('TodoListApplication', () => {
```

Comment:
IIUC, you are testing `TodoListImageRepository` in this file. Let's make that clear in the test suite name and the file name too.

- Proposed file name: `todo-list-image-repository.integration.ts`
- Proposed suite name: `describe('TodoListImageRepository'`